code/modules/admin/admin.dm

```diff
@@ -414,6 +414,24 @@ var/global/noir = 0
 			else
 				tgui_alert(usr,"You need to be at least a Primary Administrator to enable/disable shuttle calling.")
 
+		if("radio_audio_allow")
+			if(src.level >= LEVEL_MOD)
+				for(var/obj/submachine/record_player/O in by_type[/obj/submachine/record_player])
+					O.can_play_music = TRUE
+				for(var/obj/submachine/tape_deck/O in by_type[/obj/submachine/tape_deck])
+					O.can_play_tapes = TRUE
+				logTheThing("admin", usr, null, "allowed for radio music/tapes to play.")
+			else
+				tgui_alert(usr,"You need to be at least a Moderator to allow radio audio.")
+		if("radio_audio_disallow")
+			if(src.level >= LEVEL_MOD)
+				for(var/obj/submachine/record_player/O in by_type[/obj/submachine/record_player])
+					O.can_play_music = FALSE
+				for(var/obj/submachine/tape_deck/O in by_type[/obj/submachine/tape_deck])
+					O.can_play_tapes = FALSE
+				logTheThing("admin", usr, null, "disallowed for radio music/tapes to play.")
+			else
+				tgui_alert(usr,"You need to be at least a Moderator to disallow radio audio.")
```

Comment:
any reason for this to not just be a toggle, instead of two separate commands?

Reply:
I considered making it a toggle, but I then realized that it could enter in a really weird state if someone tries to varedit a Record Player/Tape Deck. I'll try to make it a toggle without depending on each individual players though.

Reply:
I made it toggleable in the new commit, but I'll leave this open to get some opinion on how I did it and how I should be doing it.